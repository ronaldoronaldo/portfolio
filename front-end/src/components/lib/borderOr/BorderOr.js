import React from 'react'
import { BorderOrBefore, BorderOrAfter, TextBorder } from './BorderOr.style'

const BorderOr = ({ style }) => {
  const styleContainer = {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
    position: 'relative'
  }

  return (
    <div style={{ ...styleContainer, ...style }}>
      <BorderOrBefore />
      <TextBorder>ou</TextBorder>
      <BorderOrAfter />
    </div>
  )
}

export default BorderOr
