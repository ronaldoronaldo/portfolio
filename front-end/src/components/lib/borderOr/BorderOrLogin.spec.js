import React from 'react'
import { shallow } from 'enzyme'
import BorderOr from './BorderOr'

describe('<BorderOr />', () => {
  it('should be render correctly', () => {
    const wrapper = shallow(<BorderOr />)
    expect(wrapper).toMatchSnapshot()
  })
})
