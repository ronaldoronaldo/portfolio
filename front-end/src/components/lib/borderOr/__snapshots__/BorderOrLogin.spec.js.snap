// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BorderOr /> should be render correctly 1`] = `ShallowWrapper {}`;
